app/src/main/java/com/mobiledevelopmentfoodapp/MainViewModel.kt

```diff
@@ -10,12 +9,12 @@ import kotlinx.coroutines.launch
 
 class MainViewModel : ViewModel() {
 
-    var restaurant : MutableLiveData<List<Food>> = MutableLiveData<List<Food>>()
```

Comment:
Immutable data should be declared as a val instead of a var. Immutability best practices can be found here. https://kotlinlang.org/docs/coding-conventions.html#immutability